src/snowflake/snowpark/relational_grouped_dataframe.py

```diff
@@ -692,6 +699,7 @@ def _function(
 
         return df
 
+    @publicapi
     def _non_empty_argument_function(
```

Comment:
ditto

Reply:
Same here, just guards us by overriding `_emit_ast` to be false if the global AST enabled flag is off.